backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt

```diff
@@ -0,0 +1,10 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Movie(
+    val id: Long,
+    val title: String,
+    val directorId: Long,
+    val genreId: Long,
+    val productionYear: Int?,
+    val rating: Float?
```

Comment:
I think it should also include some description about the movie. Check my project for inspiration

Reply:
Also I think we should include info about actors somehow. Probably separete table in DB

Reply:
Added `plot` and `actors`